src/server/scriptVersionCache.ts

```diff
@@ -17,7 +17,7 @@ namespace ts.server {
         lineText: string | undefined;
     }
 
-    export enum CharRangeSection {
+    enum CharRangeSection {

```

Comment:
Since we're no longer exporting this, we should make it `const` so that we inline the enum values.